tinyagentos/routes/coding.py

```
@@ -132,4 +158,190 @@ async def delete_workspace(request: Request, workspace_id: str):
+        if target is None:
+            continue
+
+        if untracked:
+            # Diff against empty (show full content as +lines)
+            if target.is_file():
+                try:
+                    content = target.read_text(encoding="utf-8", errors="replace")
+                except OSError:
+                    content = ""
+                lines = content.splitlines(keepends=True)
+                patch_lines = [f"+{l}" for l in lines]
```

Ruff (0.15.17)

[error] Ambiguous variable name: `l`

(E741)